projects/1.19/assets/dimensional-doors/dimdoors/lang/zh_cn.json

```
@@ -0,0 +1,426 @@
+  "text.autoconfig.dimdoors.option.limbo.hardcoreLimbo.@Tooltip": "启用时，玩家将无法通过死亡逃离维度监牢，这将使得金质次元门和永恒源质成为逃离的唯二手段。",
+  "text.autoconfig.dimdoors.option.limbo.decaySpreadChance": "衰败蔓延概率",
+  "text.autoconfig.dimdoors.option.limbo.decaySpreadChance.@Tooltip": "未完成。",
+
+  "text.autoconfig.dimdoors.category.graphics": "图像设置",
+  "text.autoconfig.dimdoors.option.graphics.highlightRiftCoreFor": "裂痕核心高亮持续时间",
+  "text.autoconfig.dimdoors.option.graphics.highlightRiftCoreFor.@Tooltip": "在裂痕旁（并非直接放置于裂痕上）放置次元门时高亮裂痕的时间。单位为毫秒。",
+  "text.autoconfig.dimdoors.option.graphics.showRiftCore": "始终高亮裂痕核心",
+  "text.autoconfig.dimdoors.option.graphics.showRiftCore.@Tooltip": "启用后，所有裂痕的核心都将以动画形式高亮。",
+  "text.autoconfig.dimdoors.option.graphics.riftSize": "裂痕渲染大小",
+  "text.autoconfig.dimdoors.option.graphics.riftSize.@Tooltip": "裂痕渲染大小倍率，默认为1。",
+  "text.autoconfig.dimdoors.option.graphics.riftJitter": "裂痕抖动强度",
+  "text.autoconfig.dimdoors.option.graphics.riftJitter.@Tooltip": "裂痕抖动强度倍率，默认为1。",
+
+  "argument.dimdoors.schematic.invalidNamespace": "未知的结构命名空间，应为%s中的一个，找到于%s。",
+  "command.dimdoors.schematicv2.unknownSchematic": "未知的结构\"%s\"于命名空间\"%s\"。",
+  "dimdoors.destination": "目标类型",
+
+  "dimdoors.advancement.root": "维度门",
+  "dimdoors.advancement.root.desc": "向着未知空间的冒险",
+  "dimdoors.advancement.dark_ostiology": "逃出生天",
+  "dimdoors.advancement.dark_ostiology.desc": "放置一个木质维度门",
+  "dimdoors.advancement.darklight": "暗光源",
+  "dimdoors.advancement.darklight.desc": "获得现实之壁",
+  "dimdoors.advancement.door_to_adventure": "冒险之门",
+  "dimdoors.advancement.door_to_adventure.desc": "进入地牢遗迹",
+  "dimdoors.advancement.enter_limbo": "地狱边境",
+  "dimdoors.advancement.enter_limbo.desc": "进入维度监牢",
+  "dimdoors.advancement.hole_in_the_sky": "虚空虫洞",
+  "dimdoors.advancement.hole_in_the_sky.desc": "进入一条裂隙",
+  "dimdoors.advancement.home_away_from_home": "离家很远的家",
+  "dimdoors.advancement.home_away_from_home.desc": "进入私人口袋维度",
+  "dimdoors.advancement.lost_and_found": "失而复得",
+  "dimdoors.advancement.lost_and_found.desc": "打开地牢遗迹中的箱子",
+  "dimdoors.advancement.out_of_time": "连接超时",
+  "dimdoors.advancement.out_of_time.desc": "试图在口袋维度设置重生点",
+  "dimdoors.advancement.public_pocket": "公用空间",
+  "dimdoors.advancement.public_pocket.desc": "进入公用口袋维度",
+  "dimdoors.advancement.string_theory": "弦理论",
+  "dimdoors.advancement.string_theory.desc": "收集一些世界纤维",
+  "dimdoors.advancement.world_unfurled": "世界崩解",
+  "dimdoors.advancement.world_unfurled.desc": "收集一些破碎之壁",
+  "dimdoors.advancement.unravelled_but_immutable": "永恒的破碎",
+  "dimdoors.advancement.unravelled_but_immutable.desc": "收集一些坚韧纤维",
+  "dimdoors.advancement.fuzzy_unreality": "模糊而虚幻",
+  "dimdoors.advancement.fuzzy_unreality.desc": "收集一些崩解的纤维",
+
+  "biome.dimdoors.black_void": "黑色虚空（公用口袋维度空间）",
+  "biome.dimdoors.dangerous_black_void": "凶险的黑暗虚空（地牢遗迹空间）",
+  "biome.dimdoors.limbo": "维度监牢",
+  "biome.dimdoors.white_void": "纯白色虚空（私人口袋维度空间）",
+
+  "limbo.death.fell.accident.ladder": "%1$s从梯子上摔了下来，并且坠入了维度监牢",
```

```suggestion
  "limbo.death.fell.accident.ladder": "%1$s从梯子上摔了下来，并坠入了维度监牢",
```
```suggestion
  "limbo.death.fell.accident.ladder": "%1$s从梯子上摔入了维度监牢",
```
总之无论怎么翻译，请保证上下统一；并参考https://minecraft.fandom.com/zh/wiki/%E6%AD%BB%E4%BA%A1%E6%B6%88%E6%81%AF